src/collector.ts

```diff
@@ -71,6 +71,73 @@ class DependencyCollector implements IDependencyCollector {
     }
 }
 
+class NaivePyParser {
+    constructor(objSam: any) {
+        this.objSam = objSam;
+        this.parser = this.createPyParser()
+    }
+
+    objSam: any;
+    parser: Xml2Object;
+    dependencies: Array<IDependency> = [];
+    isDependency: boolean = false;
+    versionStartLine: number = 0;
+    versionStartColumn: number = 0;
+
+    createPyParser(): any {
+        let deps = this.dependencies;
+        this.objSam.forEach(function(obj) {
+            let entry: IKeyValueEntry = new KeyValueEntry(obj["pkgName"], {line: 0, column: 0});
+                entry.value = new Variant(ValueType.String, obj["version"]);
+                entry.value_position = {line: obj["line"], column: obj["pos"]};
+                let dep: IDependency = new Dependency(entry);
+                deps.push(dep);
+        });
+    }
+
+     parse(): Array<IDependency> {
+        return this.dependencies;
+    }
+}
+
+let toObject = (arr:any) => {
```

Comment:
I think `requirement.txt` files can be more complex and thus this code probably won't cover everything, but it's a start.